plugins/octopus-deploy/README.md

```diff
@@ -42,9 +42,9 @@ const cicdContent = (
 )
 ```
 
-3. Add `octopus.com/project-id` annotation in catalog descriptor file
+3. Add `octopus.com/project-id` and optionally an `octopus.com/space-id` annotation in the catalog descriptor file.
```

Comment:
This is no longer the case, right

Reply:
Yeah, you're right. Not exactly sure what happened here, but I fixed the docs anyway.